src/global-firestore-data/semesters.ts

```diff
@@ -128,8 +128,12 @@ export const deleteCourseFromSemester = (
   deleteCourseFromSelectableRequirements(courseUniqueID);
 };
 
-export const deleteAllCoursesFromSemester = (season: FirestoreSemesterType, year: number): void => {
-  // TODO add gtag event
+export const deleteAllCoursesFromSemester = (
```

Comment:
Since this function is not being used, can we make sure it gets called when this functionality is implemented (maybe by adding it to a notion item)?

Reply:
It's currently being implemented by @bungaepyo, probably a notion item is unnecessary?